rskj-core/src/main/java/co/rsk/net/sync/SyncConfiguration.java

```diff
@@ -24,10 +24,10 @@
 
 @Immutable
 public final class SyncConfiguration {
-    public static final SyncConfiguration DEFAULT = new SyncConfiguration(5, 60, 30, 5, 20, 192, 20, 10);
+    public static final SyncConfiguration DEFAULT = new SyncConfiguration(5, 60, 30, 5, 20, 192, 20, 10, 0);
```

Comment:
seems it's only used in tests. `@VisibleForTesting` ?

Reply:
Addressed